.evergreen/config.yml

```diff
@@ -349,6 +349,13 @@ functions:
           ${PREPARE_SHELL}
           sh ${DRIVERS_TOOLS}/.evergreen/stop-orchestration.sh
 
+  "run OCSP responder":
+    - command: shell.exec
+      params:
+        script: |
+          ${PREPARE_SHELL}
+          TEST_COLUMN=${TEST_COLUMN} CERT_TYPE=${CERT_TYPE} USE_DELEGATE=${USE_DELEGATE} sh ${PROJECT_DIRECTORY}/.evergreen/run-ocsp-responder.sh
```

Comment:
The certificates are part of drivers-evergreen-tools, the `run-ocsp-responder` file is not. That said, some of the other files (e.g. run-orchestration.sh, orchestration config) are duplicated already and could be removed in favour of using drivers-evergreen-tools. Since this has implications for the travis build as well, I've created [PHPC-PHPC-1626](https://jira.mongodb.org/browse/PHPC-1626) to track this.